### PR TITLE
Change match[0] to matches[0] in "Searching for an organisation"

### PR DESCRIPTION
**Pull request by elisaado**
Would otherwise give "ReferenceError: match is not defined"
(I'm terrible with naming changes)

### CODE REVIEW

**Comment by AlexGustafsson** [comment]
Great catch!

Would you like to fix the root cause of the problem?

I've used [Slate](https://github.com/lord/slate) to generate the static documentation for the GitHub Pages. The code and documentation source used to generate the page is available in the [slate-branch](https://github.com/AlexGustafsson/itslearning/tree/slate). 

[The typo you found](https://github.com/AlexGustafsson/itslearning/blame/493abdba10509858a2d9f9b4c37b6635cbe0e1d0/source/includes/_itslearning.md#L16-L23) originated from the source documentation.

```javascript
ItsLearning.searchOrganisation('Blekinge Tekniska Högskola')
  .then(matches => {
    console.log(`A match was: ${match[0].name} with the id ${match[0].id}`);
  }).catch(error => {
    // there was an error with the request
  });
```

I'd rather see the snippet being changed to something like the following:

```javascript
ItsLearning.searchOrganisation('Blekinge Tekniska Högskola')
  .then(matches => {
    console.log(`A match was: ${matches[0].name} with the id ${matches[0].id}`);
  }).catch(error => {
    // there was an error with the request
  });
```

(Use of `matches` consistently. This is to ensure readability)

When the source has been changed, a re-build has to be made. Right now Linux and macOS are supported platform. See [Slate's README](https://github.com/lord/slate) for more information.

The generated site will be available in the `build` folder. The content of said folder should then be made the content of the `gh-pages` branch.

So to sum up:

1. Fix the root cause of the issue in the `slate` branch
2. Build the static site and add to `gh-pages`

**Comment by elisaado** [comment]
Yeah sure, shouldn't be too hard :D 
I'll take a look.

**Comment by AlexGustafsson** [comment]
Closing as the changes discussed were made in seperate pull requests (#4 & #5).
